cartridge/roles/vshard-storage.lua

```diff
@@ -67,12 +68,23 @@ local function init()
 
     local opts, err = require('cartridge.argparse').get_opts({
         add_vshard_storage_alerts_to_issues = 'boolean',
+        auto_disable_vshard_storage = 'boolean',
     })
     if err == nil then
         vars.enable_alerting = opts.add_vshard_storage_alerts_to_issues
+        vars.auto_disable = opts.auto_disable_vshard_storage
     end
 end
 
+local first_apply = true
+local function before_apply_config(_)
+    if first_apply or vars.auto_disable then
+        vshard.storage.disable()
+    end
+    first_apply = false
```

Comment:
Not sure it matters but in case of roles hot reload this value will be reloaded as well.